main.py

```diff
@@ -24,13 +25,17 @@
 # Store predictons
 prediction_msgs = {}
 
+trueskill_predictions = {}
+
+
 @app.route('/')
 def hello():
-    return 'Hello World!'
+    return app.send_static_file('index.html')
 
 
 @app.route('/tba-webhook', methods=['POST'])
 def tba_webhook():
+    print("prediction")
```

Comment:
Please avoid using `print()`. If you wish to log things, use `app.logger`. I suspect this was for debugging though.